generator/src/main.rs

```diff
@@ -2126,6 +2126,9 @@ fn xr_bitmask_value_name(ty: &str, name: &str) -> Ident {
     let ty = &ty[0..ty.len() - "Flags".len()];
     let prefix_len = ty.to_shouty_snake_case().len() + 1;
     let end = name.rfind("_BIT").unwrap();
+    if prefix_len == end + 1 {
+        return Ident::new(name, Span::call_site());
+    }
```

Comment:
For consistency and concision, we should generate e.g. `PASSTHROUGH_CAPABILITY` here, omitting the redundant `XR_` and `_BIT` bookends. A comment justifying the logic would also be nice.